### DISCUSSION TITLE
Wrecked Coilgun Loader still using legacy sprite

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14578] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The wrecked coilgun loader doesnt use its broken equivalent and instead uses an old legacy sprite
You can't see the ammo boxes loaded into it

![image](https://github.com/user-attachments/assets/5710f801-d92f-4546-9485-89da20b22810)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/1768fc7e7370e490b115ad6c24923ebc92932260, fix coming up in the next unstable build!
